tests/python/pants_test/backend/jvm/tasks/jvm_compile/java/test_zinc_compile_integration.py

```diff
@@ -228,30 +221,30 @@ def x(y: Option[Int] = None) = {
                 }
 
                 lib_spec, src_file_a, srcfile_content = _create_a_target_containing_two_sources()
+                with buildroot_manager.pushd():
+                    pants_run = self.run_pants_with_workdir(
+                        ["-ldebug", "compile", lib_spec],
+                        workdir=workdir,
+                        config=config,
+                        # cwd=tmp_build_root,
+                    )
+                    self.assert_success(pants_run)
+                    self.assertIn(
+                        "Full compilation, no sources in previous analysis", pants_run.stdout_data
+                    )
+                    self.assertIn("Compiling 2 Scala sources", pants_run.stdout_data)
```

Comment:
Nit: preferred to use `assert x in xs`

Reply:
my goal here was to avoid modifying this beyond ws. I could retouch this to minimize--or fix it.

Reply:
I think it'd be better to fix all the assertIn's in one pass and I feel like that's out of scope.

Reply:
Fair enough, fine to keep what we have.